clients/src/main/java/org/apache/kafka/clients/producer/internals/Sender.java

```diff
@@ -691,11 +716,11 @@ private void sendProduceRequest(long now, int destination, short acks, int timeo
         }
 
         String transactionalId = null;
-        if (transactionManager != null && transactionManager.isTransactional()) {
+        if (transactionManager != null && transactionManager.isTransactional())
```

Comment:
here and elsewhere, the code style for kafka requires that there be braces even around single line if statements like this.

Reply:
restored the curly braces.